lib/features/upload/presentation/controller/upload_controller.dart

```diff
@@ -143,6 +144,7 @@ class UploadController extends BaseController {
               leadingSVGIconColor: Colors.white,
               leadingSVGIcon: imagePaths.icInsertImage);
           }
+          consumeState(Stream.value((Left(failure))));
```

Comment:
idem

Reply:
With upload only single errors from returned events are placed in ErrorAttachmentUploadState

Reply:
idem